cloudmailer/cloudmailer.py

```diff
@@ -120,7 +120,7 @@ def getCredentials(self):
         cred = dict()
         if not 'OS_AUTH_URL' in os.environ:
                 print('Credentials not loaded to environment: did you load the rc file?')
-                exit(1)
+                sys.exit(1)
```

Comment:
This is a good opportunity to stop using the same error code for all errors, and use sequential numbers since we are changing the exit calls.

Reply:
I think that is missuing of exit codes. I think the important part is that `sys.exit(0)` is for when it successfull. Most of the exit calls except `sys.exit(2)` in `readConfiguration()`  does print a messge that should be easily understandable.

Reply:
oookkkk.....